website/docusaurus.config.js

```diff
@@ -23,17 +23,11 @@ module.exports = {
                 {
                     type: 'docsVersionDropdown',
                     position: 'left',
-                    dropdownItemsAfter: [{ to: '/versions', label: 'All versions' }],
                     dropdownActiveClassDisabled: true,
                 },
                 {
                     to: 'docs/mainnet/joining-eth2',
```

Comment:
does this need to be changed? docs/mainnet/joining-eth2

Reply:
Yep, ill do that one in the next PR